src/routes/workouts-route.php

```diff
@@ -132,8 +73,17 @@ public function register_routes() {
 	 * @return WP_REST_Response the configuration of the workouts.
 	 */
 	public function get_workouts() {
+		$workouts_option = $this->options_helper->get( 'workouts_data' );
+
+		/**
+		 * Filter: 'Yoast\WP\SEO\workouts_options' - Allows adding workouts options by the add-ons.
+		 *
+		 * @api array $workouts_option The content of the `workouts_data` option in Free.
+		 */
+		$workouts_option = \apply_filters( 'Yoast\WP\SEO\workouts_options', $workouts_option );
```

Comment:
Usually we do not add the namespace to the hookname. Did you do that intentionally?

Reply:
From what I recall this is the new recommended format for new hooks (e.g. the ones we introduced in the Social Templates feature).

Reply:
Ahh alright, good to know, thanks!